system-int-test/test/cmr/system_int_test/bootstrap/bulk_index_test.clj

```diff
@@ -233,8 +242,8 @@
       (bootstrap/bulk-index-concepts "PROV1" :collection colls)
       (bootstrap/bulk-index-concepts "PROV1" :granule [(:concept-id gran2)])
       (bootstrap/bulk-index-concepts "PROV1" :tag [(:concept-id tag1)])
-      (bootstrap/bulk-index-concepts "CMR" :access-group [(:concept-id group2)])
-      (bootstrap/bulk-index-concepts "CMR" :acl [(:concept-id acl2)])
+      ; (bootstrap/bulk-index-concepts "CMR" :access-group [(:concept-id group2)])
```

Comment:
We should put a comment back in explaining why any of this code is commented out.